BeagleBone_black/src/navigation/main.hpp

```diff
@@ -28,6 +28,7 @@
 namespace hyped {
 
 using data::Sensors;
+using data::State;
```

Comment:
if you are not using this in the header, consider moving the import into .cpp file